### DISCUSSION TITLE
QRouteTabs active path

### DISCUSSION THREAD

**Discussion by lisadejonghe** [quasarframework/quasar#13111] [General - Components / Directives / etc]
I have the following QTabs

```
      <q-tabs class="absolute-center" v-model="tab" shrink>
        <q-route-tab name="students" label="Select Students" to="/students" exact />
        <q-route-tab name="projects" label="Projects" to="/projects" exact />
        <q-route-tab name="users" label="Manage Users" to="/users" exact />
      </q-tabs>
```

When on the `/students` page, the QTab that is active is underlined (active-class). On this page, I can click students, but then the link changes to `/students/:id`. Problem now is that, when the link changes, the tab is not active anymore, but it should be. How can I fix this?

**Comment by MilosPaunovic** [comment]
Remove the `exact` prop.

**Comment by lisadejonghe** [comment]
That doesn't work for me. I already thought of that.

**Comment by Gnopps** [comment]
I have the exact same problem. q-route-tab is active when using child routing but not when using /:parameter. Did you ever get this solved @lisadejonghe?

**Comment by lisadejonghe** [comment]
@gnopps No, I'm sorry.

**Reply by Gnopps** [reply]
Thanks for your quick reply. I submitted this as a bug report now: https://github.com/quasarframework/quasar/issues/13931

**Reply by lisadejonghe** [reply]
Thanks!
